app/job_test.go

```diff
@@ -23,11 +24,9 @@ func TestGetJob(t *testing.T) {
 
 	defer th.App.Srv.Store.Job().Delete(status.Id)
 
-	if received, err := th.App.GetJob(status.Id); err != nil {
-		t.Fatal(err)
-	} else if received.Id != status.Id || received.Status != status.Status {
-		t.Fatal("inccorrect job status received")
-	}
+	received, err := th.App.GetJob(status.Id)
+	require.Nil(t, err)
+	assert.Equal(t, status, received, "incorrect job status received")
```

Comment:
nit: you can use `require.Equal` here, as the previous `t.Fatal` would have stopped the execution as `require` will, and that way you would not need to import `assert`

Reply:
You're right, it's not iso-functional. Nevertheless, I did it on purpose, inspired by what was done in other parts (e.g. [app/session_test.go](https://github.com/mattermost/mattermost-server/blob/c19bddb51bfbb51c05303e6ca32fff5ec425b12a/app/session_test.go#L64)) where execution is stopped only when they would be error if it continues. 
I presume it's not part of this PR to change behaviour, so I will commit a fix. Thanks!

Reply:
done